JsonPath/Expressions/Operators.cs

```diff
@@ -83,22 +83,22 @@ public static bool TryParse(ReadOnlySpan<char> source, ref int index, [NotNullWh
 
 		var portion = source[index..(index + 2)];
 
-		if (portion.Equals("==", StringComparison.Ordinal))
+		if (portion.Equals("==".AsSpan(), StringComparison.Ordinal))
```

Comment:
Since these are constants, we should store the span representation in a field so it doesn't have to be recalculated repeatedly.

Reply:
Resolved